nubis/puppet/apache.pp

```diff
@@ -59,6 +59,8 @@
     "set X-Nubis-Version ${project_version}",
     "set X-Nubis-Project ${project_name}",
     "set X-Nubis-Build   ${packer_build_name}",
+    'set Strict-Transport-Security "max-age=31536000"',
```

Comment:
Done with the application itself.

Reply:
@joekrejci Gozer is saying that the django app can set this header itself. The process is described in the `README` (near the bottom), and he enabled the feature in a commit earlier today (https://github.com/mozilla-itcloud/air.mozilla.org/commit/10496f106af7e1532a841e53793713a18ef3f9dd). So you can remove this line.